packs/aws/sensors/sqs_sensor.py

```diff
@@ -57,11 +57,13 @@ def setup(self):
 
     def poll(self):
         for queue in self.input_queues:
-            msg = self._receive_messages(queue=self._get_queue_by_name(queue))
-            if msg:
-                payload = {"queue": queue, "body": msg[0].body}
-                self._sensor_service.dispatch(trigger="aws.sqs_new_message", payload=payload)
-                msg[0].delete()
+            msgs = self._receive_messages(queue=self._get_queue_by_name(queue),
+                                          num_messages=self.max_number_of_messages)
+            for msg in msgs:
+                if msg:
```

Comment:
Actually, NVM, I see that the code before hand did the same, you just added support for multiple messages...